torch/fx/passes/utils/matcher_utils.py

```diff
@@ -172,7 +179,14 @@ def _match_nodes(self, pn: Node, gn: Node, match: InternalMatch) -> bool:
         pn_flatten_args, _ = pytree.tree_flatten(pn.args)
         gn_flatten_args, _ = pytree.tree_flatten(gn.args)
 
-        if len(pn_flatten_args) == len(gn_flatten_args):
+        if pn.kwargs.keys() == gn.kwargs.keys():
```

Comment:
nit: do we need to make sure the keys are the same as well?

Reply:
pn.kwargs.keys() == gn.kwargs.keys() 

hmm, is something missing with this check?